api/specs/webserver/openapi-catalog.yaml

```diff
@@ -120,6 +120,286 @@ paths:
         default:
           $ref: "./openapi.yaml#/components/responses/DefaultErrorResponse"
 
+  catalog_services_service_key_service_version_inputs:
+    get:
+      tags:
+        - catalog
+      operationId: list_service_inputs_handler
+      parameters:
+        - in: path
+          name: service_key
+          required: true
+          schema:
+            pattern: ^(simcore)/(services)/(comp|dynamic|frontend)(/[\w/-]+)+$
```

Comment:
this does not fit with the above pattern... frontend is here. and the \w... something is not consistent

Reply:
I found that in the webserver, we are using two different regex patterns for ``service_key`` having different endings, namely:

1. ``^(simcore)/(services)/(comp|dynamic|frontend)(/[^\s/]+)+$`` (this variant sometimes frontend is not included when path refers to "running services")
2. ``^(simcore)/(services)/(comp|dynamic|frontend)(/[\w/-]+)+$``

pattern 2. is included in model-services (search for ``SERVICE_KEY_RE``)  and pattern 1 is the old one.

I checked both patterns against all the names that we have in master registry in ``packages/models-library/tests/test_services.py::test_service_key_regex_pattern`` and both work fine.

For consistency I set everything with pattern 2